### PR TITLE
API for Recovery codes

### PR DESCRIPTION
**Pull request by bguyza**

### CODE REVIEW

File: site/_layouts/doc.liquid [hunk 1]
```diff
@@ -115,6 +115,7 @@
                     <li {% if page.url == "/docs/v1/tech/apis/lambdas/" %}class="active"{% endif %}><a href="/docs/v1/tech/apis/lambdas/">Lambdas</a></li>
                     <li {% if page.url == "/docs/v1/tech/apis/login/" %}class="active"{% endif %}><a href="/docs/v1/tech/apis/login/">Login</a></li>
                     <li {% if page.url == "/docs/v1/tech/apis/passwordless/" %}class="active"{% endif %}><a href="/docs/v1/tech/apis/passwordless/">Passwordless</a></li>
+                    <li {% if page.url == "/docs/v1/tech/apis/recovery-codes/" %}class="active"{% endif %}><a href="/docs/v1/tech/apis/recovery-codes/">Recovery Codes</a></li>
```
Comment (mooreds) [inline site/_layouts/doc.liquid:118]: Please also add site/docs/v1/tech/apis/index.adoc

**Review by mooreds — CHANGES_REQUESTED** [review]
One small change, other than that, looks good to me!

File: site/docs/v1/tech/apis/recovery-codes.adoc [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+==== Request Parameters
+
+[.api]
+[field]#userId# [type]#[UUID]# [required]#Required#::
+The Id of the User to retrieve Recovery Codes for.
+
+=== Response
+
+The response for this API contains the Recovery Codes that are assigned to the User
```
Comment (robotdan) [inline site/docs/v1/tech/apis/recovery-codes.adoc:77]: The response will contain the remaining recovery codes, each time one is used it is removed, so this response will contain between 0 and 10 codes.

File: site/_layouts/doc.liquid (hunk 1 above)
Comment (mooreds) [reply site/_layouts/doc.liquid:118]: Ah, that makes total sense to me. To avoid merge issues, might make sense to do this after #578 is merged.

Comment (bguyza) [reply site/_layouts/doc.liquid:118]: Ah yes, I originally thought about this but wanted to get things moving.  My brain was in a complete fog last week so disregard any questionable decisions I might have made :)

**Pull request closed without merging**